ci/tasks/light-aws/test-drivers.yml

```
@@ -0,0 +1,23 @@
+---
+platform: linux
+
+inputs:
+- name: builder-src
+- name: bosh-stemcells-ci
+run:
+  path: bosh-stemcells-ci/ci/tasks/light-aws/test-drivers.sh
+params:
+  aws_account_id:                        ""
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)